packages/backend-defaults/src/entrypoints/cache/CacheManager.ts

```diff
@@ -157,7 +157,7 @@ export class CacheManager {
         ttl: defaultTtl,
         store,
         emitErrors: false,
-        useRedisSets: this.useRedisSets,
+        useKeyPrefix: false,
```

Comment:
It's quiet confusing, but this option disables appending additional keyprefix. Without this we get doubled prefix (e.g. techdocs:techdocs:<key>).

This is described here:
- https://github.com/jaredwray/keyv/tree/main/packages/redis#migrating-from-v3-to-v4.

---

Also, the same configuration causes issues for memcache implementation (it has doubled prefix), but since the same happens on master I did not want to change it in this MR.